### PR TITLE
Blocking a physical copy of the game upon purchase

### PR DESCRIPTION
**Pull request by denis-maslovskiy**
Implemented blocking the game for 15 minutes from the date of purchase
Created custom Timer which show user remaining time
Small fixes in different places of project

### CODE REVIEW

File: client/src/pages/Basket.js [hunk 1]
```diff
@@ -12,11 +12,38 @@ import {
...
 import { updateGameData } from "../redux/games/gamesActions";
-import { getGameInfo } from '../helpers/gameHelpers';
+import { getGameInfo } from "../helpers/gameHelpers";
 import image from "../img/3.jpg";
 import "../styles/basket.scss";
 
-const itemsInBasketList = (itemsInBasket, removeGameHandler) => {
+const Timer = ({ removeGameHandler, gameName, dateAddedToBasket }) => {
+  let [time, setTime] = useState(
+    900 - Math.floor((Date.now() - Date.parse(dateAddedToBasket)) / 1000)
```
Comment (MeleshkoYuliya) [inline client/src/pages/Basket.js:21]: Why `let`? Why `900 - ...`

File: client/src/pages/Basket.js [hunk 2]
```diff
@@ -12,11 +12,38 @@ import {
...
+  );
+  
+  const getMinutes = () => {
+    return ("0" + Math.floor((time % 3600) / 60)).slice(-2);
+  };
+
+  const getSeconds = () => {
+    return ("0" + (time % 60)).slice(-2);
+  };
```
Comment (MeleshkoYuliya) [inline client/src/pages/Basket.js:30]: For get seconds and get minutes you can use `moment js` or new Date api. For time i think it is better to use UTC format for example. In which format is `time`?

File: client/src/pages/Basket.js (hunk 1 above)
Comment (denis-maslovskiy) [reply client/src/pages/Basket.js:21]: Replaced let with const
900 = 15 minut

File: client/src/pages/Basket.js [hunk 3]
```diff
@@ -2,21 +2,39 @@ import React, { useState, useEffect } from "react";
...
+
+  const [minutes, setMinutes] = useState();
+  const [seconds, setSeconds] = useState();
+
+  useEffect(() => {
+    const timer = setInterval(() => {
+      let time = new Date(new Date(fifteenMinutes) - new Date(Date.now() - Date.parse(dateAddedToBasket)));
+      setMinutes(time.getMinutes());
+      setSeconds(time.getSeconds());
+      if (time.getMinutes() === 0 && time.getSeconds() === 0 || time.getMinutes() > fifteenMinutes / 60000) {
```
Comment (MeleshkoYuliya) [inline client/src/pages/Basket.js:23]: move `60000` to the constant

**Pull request merged**